libs/pulumi/src/generators/init/generator.ts

```diff
@@ -112,10 +120,10 @@ export default async function (host: Tree, options: PulumiGeneratorSchema) {
         normalizedOptions.targetProjectName,
     )
     targetProjectConfig.targets = targetProjectConfig.targets || {}
-    targetProjectConfig.targets.up = {
-        executor: '@wanews/nx-pulumi:up',
+    targetProjectConfig.targets.deploy = {
+        executor: '@nrwl/workspace:run-commands',
         options: {
-            targetProjectName: normalizedOptions.targetProjectName,
+            commands: [`nx run ${normalizedOptions.projectName}:up`],
```

Comment:
this is the simple option, could also create a custom executor and do it in-process, but this is less code.